samples/java-protobuf-eventsourced-customer-registry-subscriber/src/main/proto/customer/view/customer_view.proto

```diff
@@ -33,32 +41,25 @@ service AllCustomersView {
 
   // consume events published by java-protobuf-eventsourced-customer-registry/CustomerEventsServiceAction
   option (kalix.service).eventing.in.direct = { // <1>
-    // end::view[]
-    // Normally the Kalix service deployment name
-    // (but host.docker.internal + hardcoded port here to be able to run both locally with docker)
-    service: "host.docker.internal:9000"
-    /*
-    // tag::view[]
     service: "customer-registry" // <2>
-    // end::view[]
-     */
-    // tag::view[]
     event_stream_id: "customer_events" // <3>
   };
 
-  rpc ProcessCustomerCreated(api.Created) returns (api.Customer) { // <4>
+  rpc ProcessCustomerCreated(api.Created) returns (Customer) { // <4>
+    option (kalix.method).view.update.transform_updates = true;
```

Comment:
To clarify, if the input is different from the output, then the flag should not be required because it's obviously a transformation.

Reply:
Hmm, just noticed that in CI we are compiling the proto with `io.kalix:kalix-maven-plugin:1.2.0:generate`. 

This is not correct. It should compile with the current snapshot. 

It works on my machine because I was using a snapshot. 

On the other hand, it's not clear why it was not failing before. This is a compile time error and, although this sample was not being tested, it was being compiled.

Reply:
Let me check why it ended up not using the correct version.